libs/community/langchain_community/agent_toolkits/sql/prompt.py

```diff
@@ -4,6 +4,9 @@
 Given an input question, create a syntactically correct {dialect} query to run, then look at the results of the query and return the answer.
 Unless the user specifies a specific number of examples they wish to obtain, always limit your query to at most {top_k} results.
 You can order the results by a relevant column to return the most interesting examples in the database.
+You can filter the results by a relevant column to return the most relevant examples in the database.
```

Comment:
would prefer not to update default prompts since that silently changes behavior of user code, and these can all be directly specified by the user if needed. would prefer that we update the [SQL prompting strategies](https://python.langchain.com/docs/use_cases/sql/prompting) docs to suggest this approach